i18n-slf4j/pom.xml

```diff
@@ -35,6 +36,17 @@
   <name>ForgeRock I18N SLF4J Support</name>
   <description>This module provides an interface for efficiently writing localized messages out to a SLF4J Logger</description>
   <packaging>bundle</packaging>
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>jp.openam.commons</groupId>
+        <artifactId>forgerock-bom</artifactId>
```

Comment:
Is it necessary to import BOM in this child project?

Reply:
Thank you for the review.
I deleted this part because it is unnecessary.